windows/shared/logging/PlogTraceLogging/PlogTraceLogging.cxx

```diff
@@ -0,0 +1,13 @@
+
+#include <chrono>
+#include <iomanip>
+#include <sstream>
+
+#include <PlogTraceLogging/PlogTraceLogging.hxx>
```

Comment:
None of these should be needed here, can they be removed? The only `#include`s in a header should be those that define types used in the header. Sometimes for legacy reasons, an exception to this guideline is made for the windows types which are declared in `windows.h` but the order of inclusion of this header can affect other code, so is required to be explicitly included by the caller instead.

In this case, this header doesn't define anything which uses types from `chrono`, `iomanip`, nor `sstream`.

Also, the header that defines `TRACELOGGING_DEFINE_PROVIDER` (`TraceLoggingProvider.h`?) seems to be missing, so that should be added. This also requires `windows.h`, so that needs to be added as well.

```suggestion
#include <windows.h>

#include <TraceLoggingProvider.h>
```

Reply:
The PlogTraceLogging.hxx header file provides the windows.h and TraceLoggingProvider.h headers